apps/accounts/templates/account/email_confirm.html

```diff
@@ -6,35 +6,54 @@
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.1/css/materialize.min.css">
 </head>
 
-<div class="dyn-header padding-lr-header">
-  <nav>
-    <div class="nav-wrapper">
-
-    <ul class="left">
-      <li><a href="https://evalai.cloudcv.org/" class="waves-effect waves-dark">Home</a></li>
-    </ul>
-      <ul class="right">
-        <li><a href="https://evalai.cloudcv.org/auth/signup" class="waves-effect waves-dark">Sign Up</a></li>
-        <li><a href="https://evalai.cloudcv.org/auth/login" class="waves-effect waves-dark">Log In</a></li>
-      </ul>
-
-    </div>
-  </nav>
-</div>
-<section class="bg-section">
-    <div class="bg-image"></div>
-</section>
-<section class="ev-container auth-container center">
-  <div class="ev-head-small">
-    <div class="row">
-      <div class="col s12 m4 offset-m4 text-highlight white-text center w-300">
-        Your email has been verified successfully.
-      </div>
+<!-- loader init -->
+<eval-loader></eval-loader>
+<!--auth parent view -->
+<section class="auth-container ">
+    <!-- graphic animated view -->
+    <div class="graphic-cont">
+        <img class="star-img star1" src="{% static 'accounts/images/star1.png' %}">
+        <img class="star-img star2" src="{% static 'accounts/images/star2.png' %}">
+        <div class="moon">
+            <img src="{% static 'accounts/images/moon.png' %}">
+        </div>
+        <div class="sun">
+            <img src="{% static 'accounts/images/sun.png' %}">
+        </div>
+        <div class="auth-cloud-cont">
+            <img src="{% static 'accounts/images/auth-cloud.png' %}">
+        </div>
+        <div class="ufo">
+            <img src="{% static 'accounts/images/spaceman.png' %}">
+        </div>
     </div>
-    <div class="row">
-      <div class="col s12 m4 offset-m4 text-highlight center w-300">
-        <a href="https://evalai.cloudcv.org/auth/login" class="waves-effect waves-dark btn ev-btn-dark blue-grey darken-4">Log In</a>
-      </div>
+    <!-- rendering auth children here -->
+    <div class="auth-cont">
+        <div class="row">
+            <div class="col s12 m6 offset-m3">
+                <div class="left">
+                    <h5><strong><a ui-sref="home" class="dark-link"><img class="auth-logo" src="{% static 'accounts/images/evalai-logo.png' %}"></a></strong></h5>
```

Comment:
@Ram81 Can we please align logo and below text with the text `Your email has been verified successfully.`
@deshraj What do you say /

Reply:
Sure @RishabhJain2018